### PR TITLE
Improvements to target-search

### PR DESCRIPTION
**Pull request by cquiroz**
This fixes a bug where sometimes the call to do simbad search fails. It was solved by adding a retry as in other cases

Some UI fixes too:
* Properly square the go to target button in aladin
* Better positioning of the search icon
* Animate when search is in progress
* Better position of the search error tooltip

### CODE REVIEW

File: common/src/main/scala/explore/common/RetryHelpers.scala [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import retry.RetryDetails._
+import retry.RetryPolicies._
+import retry._
+
+import java.{ util => ju }
+import scala.concurrent.duration._
+
+import ju.concurrent.TimeUnit
+
+trait RetryHelpers {
```
Comment (rpiaggio) [inline common/src/main/scala/explore/common/RetryHelpers.scala:17]: Nice!

**Review by rpiaggio — APPROVED** [review]
LGTM

**Comment by rpiaggio** [comment]
I wonder if we want to use the retry logic in all other places. Eg: fetching the spectroscopy matrix.

**Comment by cquiroz** [comment]
Probably we should

**Pull request merged**